src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -424,12 +425,25 @@ namespace service_nodes
       return true;
     }
 
-    vvc = {};
-    std::shared_ptr<const testing_quorum> quorum = m_core.get_testing_quorum(vote.type, vote.block_height);
```

Comment:
Just to confirm my understanding of this: does `vote.block_height` carry the non-offset height?  E.g. I vote for block 1000, using the checkpointing quorum calculated from the block at height 9989?

Reply:
Yes non-offset height, so 1000-11 = 989.